.github/workflows/actionlint.yml

```diff
@@ -0,0 +1,49 @@
+name: actionlint
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/*.yml'
+      - '.github/workflows/*.yaml'
+  pull_request:
+    paths:
+      - '.github/workflows/*.yml'
+      - '.github/workflows/*.yaml'
```

Comment:
I think this might mean if you open a PR on a non-fork branch this will run twice? No big deal but perhaps worth a post-merge test.